tests/manage/rgw/test_host_node_failure.py

```diff
@@ -0,0 +1,96 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.resources.pod import (
+    get_rgw_pods, get_pod_node, get_noobaa_pods, wait_for_storage_pods
+)
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework.testlib import (
+    ManageTest, tier4a
+)
+from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.helpers.helpers import wait_for_resource_state
+from ocs_ci.ocs.node import get_node_objs
+
+log = logging.getLogger(__name__)
+
+
+@tier4a
+@pytest.mark.polarion_id("OCS-2374")
+@pytest.mark.bugzilla('1852983')
+class TestRGWAndNoobaaDBHostNodeFailure(ManageTest):
+    """
+    Test to verify fail node hosting
+    RGW pods and Noobaa-db pods and its impact
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+
+        """
+        self.sanity_helpers = Sanity()
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request, nodes):
```

Comment:
node_restart_teardown fixture is present, you can use that instead.

Reply:
done